vendor/modules.txt

```diff
@@ -588,7 +588,9 @@ golang.org/x/net/http/httpproxy
 golang.org/x/net/http2
 golang.org/x/net/http2/hpack
 golang.org/x/net/idna
+golang.org/x/net/internal/socks
```

Comment:
silly ask: no trying to reflect this chance in go.mod+go.sum files as well?

Reply:
We are already pulling in `golang.org/x/net` which I think is why my changes are not introducing changes to go modules.